cli.go

```diff
@@ -129,7 +135,11 @@ func (app *App) PrintUsage() {
 
 	w := tabwriter.NewWriter(app.ErrorWriter, 0, 8, 4, ' ', 0)
 	for _, name := range names {
-		fmt.Fprintf(w, "    %s\t%s\n", name, app.Commands[name].Short)
+		n := name
+		if name == mainCmd {
+			n = "[main]"
```

Comment:
Wondering how we should describe this. "main" does not seem obvious to the users, so maybe "<no command>"?